res/css/views/rooms/_RoomSublist.scss

```diff
@@ -41,7 +41,9 @@ limitations under the License.
         // The combined height must be set in the LeftPanel component for sticky headers
         // to work correctly.
         padding-bottom: 8px;
-        height: 24px;
+        // Allow the container to collapse on itself if its children
+        // are not in the normal document flow
+        max-height: 24px;
```

Comment:
This has regressed the appearance of the "Create a room" buttons when the room list is collapsed:

![Screenshot_2021-04-22 Element mx-puppet-groupme](https://user-images.githubusercontent.com/48614497/115761609-83edda80-a370-11eb-804b-cf6cbc9b268e.png)

Reply:
Hovering around also causes some weirdness 
![image](https://user-images.githubusercontent.com/2403652/116050508-78036080-a66f-11eb-9885-8c7eca6b0573.png)
I was hovering on the lower `...` and another `...` came up but in a place where I could not click on it

Reply:
![image](https://user-images.githubusercontent.com/2403652/116050619-99fce300-a66f-11eb-8cae-ee8060f72dfd.png)

Reply:
Oh thank you for the heads up, I definitely missed that one
@robintown is there already an issue created?

Reply:
@gsouquet No, I can make one